examples/workflow/3_perform_fit.ipynb

```diff
@@ -0,0 +1,244 @@
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    ".. note::\n",
+    "    We first load the relevant data from the previous steps."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "import yaml\n",
+    "\n",
+    "from tensorwaves.physics.helicityformalism.kinematics import HelicityKinematics\n",
+    "from tensorwaves.physics.particle import extract_particles\n",
+    "from tensorwaves.physics.helicityformalism.amplitude import IntensityBuilder"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open(\"amplitude_model_helicity.yml\") as input_file:\n",
+    "    recipe = yaml.load(input_file.read(), Loader=yaml.SafeLoader)\n",
+    "kinematics = HelicityKinematics.from_recipe(recipe)\n",
+    "particles = extract_particles(recipe)\n",
+    "data_sample = np.load(\"data_sample.npy\")\n",
+    "phsp_sample = np.load(\"phsp_sample.npy\")\n",
+    "intensity_builder = IntensityBuilder(particles, kinematics, phsp_sample)\n",
+    "intensity = intensity_builder.create_intensity(recipe)\n",
+    "phsp_set = kinematics.convert(phsp_sample)\n",
+    "data_set = kinematics.convert(data_sample)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.1 Define estimator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To perform a fit, you need to define an **estimator**. An estimator is a measure for the discrepancy between the intensity model and the data distribution to which you fit. In PWA, we usually use an *unbinned negative log likelihood estimator*, which can be created as follows with the `tensorwaves.estimator` module:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from tensorwaves.estimator import UnbinnedNLL\n",
+    "\n",
+    "estimator = UnbinnedNLL(intensity, data_set)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.2 Modify parameters"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "The parameters that the `.estimator.UnbinnedNLL` carries, have been taken from the `.IntensityTF` object and you'll recognise them from the YAML recipe file that we created in :doc:`Step 1 <1_create_model>`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from copy import deepcopy\n",
+    "\n",
+    "initial_parameters = deepcopy(estimator.parameters)\n",
```

Comment:
why you make a deepcopy here?

Reply:
Oh oops yes that was a debugging attempt